### DISCUSSION TITLE
1T的SSD最多可以同时P4个图吗？

### DISCUSSION THREAD

**Discussion by love-chia** [Chia-Network/chia-blockchain#2482] [Ideas]
我一直都是P3个图，但是今天我发现硬盘还有300多G 的空间，有没有用1T SSD 同时P4个图的呢！

**Comment by lishan148** [comment]
1.1.1 版本可以，不过这个版本P图效率好像变慢了

**Comment by cccat6** [comment]
800G=3
1T=4
1.6T=7
1.8T=8
2T=9

**Reply by whaidushu** [reply]
会缓存不够的吧…不是1个图都要300多g吗？我知道可以设置间隔…但我设20min间隔好像也会报错

**Reply by yaozleo** [reply]
12.8TB是否就等于50呢？
请问单块大容量SSD有没有一个上限？

**Reply by cccat6** [reply]
> 会缓存不够的吧…不是1个图都要300多g吗？我知道可以设置间隔…但我设20min间隔好像也会报错

20min太短了，起码40min起。我一般都是从1h开始慢慢调整的。

**Reply by cccat6** [reply]
> 12.8TB是否就等于50呢？
> 请问单块大容量SSD有没有一个上限？

你那是固态RAID吧？4*3.2T？理论上如果带宽足够，可以同时P50个。但是50个需要的读写要过万MBPS了，你得确认你的盘能达到这个速度。并且这个一定是用RAID卡，而不能软件RAID。以及你至少需要一个7502P或者3975WX以上的处理器，内存插128G。CPU差了的话，P起来更慢。以及我说的是“至少”，你还可以选更多核心和更高主频的CPU。古董不用考虑，效率差了无数倍。
单块上限一般不那么容易触及。至少现在3.2T的英特尔SATA SSD带宽还不至于拉垮整体速度。

**Reply by yaozleo** [reply]
> > 12.8TB是否就等于50呢？
> > 请问单块大容量SSD有没有一个上限？
> 
> 你那是固态RAID吧？4*3.2T？理论上如果带宽足够，可以同时P50个。但是50个需要的读写要过万MBPS了，你得确认你的盘能达到这个速度。并且这个一定是用RAID卡，而不能软件RAID。以及你至少需要一个7502P或者3975WX以上的处理器，内存插128G。CPU差了的话，P起来更慢。以及我说的是“至少”，你还可以选更多核心和更高主频的CPU。古董不用考虑，效率差了无数倍。
> 单块上限一般不那么容易触及。至少现在3.2T的英特尔SATA SSD带宽还不至于拉垮整体速度。

谢谢您的及时回复，我买的是镁光9300max，单块NVMe 12.8TB，并不是raid是不是就没法做到同时plot 50了。。。

**Reply by cccat6** [reply]
> > > 12.8TB是否就等于50呢？
> > > 请问单块大容量SSD有没有一个上限？
> > 
> > 
> > 你那是固态RAID吧？4*3.2T？理论上如果带宽足够，可以同时P50个。但是50个需要的读写要过万MBPS了，你得确认你的盘能达到这个速度。并且这个一定是用RAID卡，而不能软件RAID。以及你至少需要一个7502P或者3975WX以上的处理器，内存插128G。CPU差了的话，P起来更慢。以及我说的是“至少”，你还可以选更多核心和更高主频的CPU。古董不用考虑，效率差了无数倍。
> > 单块上限一般不那么容易触及。至少现在3.2T的英特尔SATA SSD带宽还不至于拉垮整体速度。
> 
> 谢谢您的及时回复，我买的是镁光9300max，单块NVMe 12.8TB，并不是raid是不是就没法做到同时plot 50了。。。

对不起，是我想岔了。我没想到你这么有钱……
读写速度相对RAID来说弱了一点，但只是效率没那么高，同时P50个还是可以做到的。

**Reply by cccat6** [reply]
建议先上机试一下，我也没搞过这么高级的SSD。
你直接同时开始创建30个并行任务，延迟1分钟，直接开始。看一下需要多少时间能全部完成。然后把这个时间除以你要同时P的数量，大概就是每个任务的间隔时间。
肯定有误差，但是对于你这么大的盘来说，本身读写就要很久了，做精细测量很费时间，就这样简单测一下好了。

**Reply by yaozleo** [reply]
> > > > 12.8TB是否就等于50呢？
> > > > 请问单块大容量SSD有没有一个上限？
> > > 
> > > 
> > > 你那是固态RAID吧？4*3.2T？理论上如果带宽足够，可以同时P50个。但是50个需要的读写要过万MBPS了，你得确认你的盘能达到这个速度。并且这个一定是用RAID卡，而不能软件RAID。以及你至少需要一个7502P或者3975WX以上的处理器，内存插128G。CPU差了的话，P起来更慢。以及我说的是“至少”，你还可以选更多核心和更高主频的CPU。古董不用考虑，效率差了无数倍。
> > > 单块上限一般不那么容易触及。至少现在3.2T的英特尔SATA SSD带宽还不至于拉垮整体速度。
> > 
> > 
> > 谢谢您的及时回复，我买的是镁光9300max，单块NVMe 12.8TB，并不是raid是不是就没法做到同时plot 50了。。。
> 
> 对不起，是我想岔了。我没想到你这么有钱……
> 读写速度相对RAID来说弱了一点，但只是效率没那么高，同时P50个还是可以做到的。

我只是为了plotting才买的，感觉fomo了。。。这快虽然tbw超高70多pb，但是读写均只有3500 MB/S，12.8容量上正好50个plot，但是如果只这一块SSD做temp drive，难道是3500/50=每个plot分到的速度？

我是在涨价前买的，现在也不想退了，只想怎么样能最佳的利用上。

**Reply by cccat6** [reply]
> > > > > 12.8TB是否就等于50呢？
> > > > > 请问单块大容量SSD有没有一个上限？
> > > > 
> > > > 
> > > > 你那是固态RAID吧？4*3.2T？理论上如果带宽足够，可以同时P50个。但是50个需要的读写要过万MBPS了，你得确认你的盘能达到这个速度。并且这个一定是用RAID卡，而不能软件RAID。以及你至少需要一个7502P或者3975WX以上的处理器，内存插128G。CPU差了的话，P起来更慢。以及我说的是“至少”，你还可以选更多核心和更高主频的CPU。古董不用考虑，效率差了无数倍。
> > > > 单块上限一般不那么容易触及。至少现在3.2T的英特尔SATA SSD带宽还不至于拉垮整体速度。
> > > 
> > > 
> > > 谢谢您的及时回复，我买的是镁光9300max，单块NVMe 12.8TB，并不是raid是不是就没法做到同时plot 50了。。。
> > 
> > 
> > 对不起，是我想岔了。我没想到你这么有钱……
> > 读写速度相对RAID来说弱了一点，但只是效率没那么高，同时P50个还是可以做到的。
> 
> 我只是为了plotting才买的，感觉fomo了。。。这快虽然tbw超高70多pb，但是读写均只有3500 MB/S，12.8容量上正好50个plot，但是如果只这一块SSD做temp drive，难道是3500/50=每个plot分到的速度？
> 
> 我是在涨价前买的，现在也不想退了，只想怎么样能最佳的利用上。

一块SSD同时P多少，有几个关键点。
1. 用作缓存的容量能给几个任务用
2. 读写速度会不会成为P盘速度的瓶颈

一般来说，读写速度不会是问题。但在你这个情况下，读写速度肯定是问题，因为容量太大了。
我P的时候，按照上面给出的 容量-P盘数 数据正常是P不了的。但它所谓的缓存占用是“峰值”占用，所以我只需要削峰填谷就能P更多。整个优化思想也是尽可能的让任务填满所有SSD空间不浪费。一般利用率都能到90%左右。
但是你这个容量太大了，我甚至怀疑等到三四十个任务启动了以后，你的读写速度就持续占满了。这个时候增加更多任务只会降低速度。
也许你可以考虑P大于k32的盘，虽然大于k32不会获得更多收益，平均效率反而会下降。但说不定适合你这种超大容量SSD。你可以试试同时P6个k35？能比较好的利用你的SSD空间。

**Reply by yaozleo** [reply]
> > > > > > 12.8TB是否就等于50呢？
> > > > > > 请问单块大容量SSD有没有一个上限？
> > > > > 
> > > > > 
> > > > > 你那是固态RAID吧？4*3.2T？理论上如果带宽足够，可以同时P50个。但是50个需要的读写要过万MBPS了，你得确认你的盘能达到这个速度。并且这个一定是用RAID卡，而不能软件RAID。以及你至少需要一个7502P或者3975WX以上的处理器，内存插128G。CPU差了的话，P起来更慢。以及我说的是“至少”，你还可以选更多核心和更高主频的CPU。古董不用考虑，效率差了无数倍。
> > > > > 单块上限一般不那么容易触及。至少现在3.2T的英特尔SATA SSD带宽还不至于拉垮整体速度。
> > > > 
> > > > 
> > > > 谢谢您的及时回复，我买的是镁光9300max，单块NVMe 12.8TB，并不是raid是不是就没法做到同时plot 50了。。。
> > > 
> > > 
> > > 对不起，是我想岔了。我没想到你这么有钱……
> > > 读写速度相对RAID来说弱了一点，但只是效率没那么高，同时P50个还是可以做到的。
> > 
> > 
> > 我只是为了plotting才买的，感觉fomo了。。。这快虽然tbw超高70多pb，但是读写均只有3500 MB/S，12.8容量上正好50个plot，但是如果只这一块SSD做temp drive，难道是3500/50=每个plot分到的速度？
> > 我是在涨价前买的，现在也不想退了，只想怎么样能最佳的利用上。
> 
> 一块SSD同时P多少，有几个关键点。
> 
> 1. 用作缓存的容量能给几个任务用
> 2. 读写速度会不会成为P盘速度的瓶颈
> 
> 一般来说，读写速度不会是问题。但在你这个情况下，读写速度肯定是问题，因为容量太大了。
> 我P的时候，按照上面给出的 容量-P盘数 数据正常是P不了的。但它所谓的缓存占用是“峰值”占用，所以我只需要削峰填谷就能P更多。整个优化思想也是尽可能的让任务填满所有SSD空间不浪费。一般利用率都能到90%左右。
> 但是你这个容量太大了，我甚至怀疑等到三四十个任务启动了以后，你的读写速度就持续占满了。这个时候增加更多任务只会降低速度。
> 也许你可以考虑P大于k32的盘，虽然大于k32不会获得更多收益，平均效率反而会下降。但说不定适合你这种超大容量SSD。你可以试试同时P6个k35？能比较好的利用你的SSD空间。

能同时开荒30个plot也算值了，按照这个tbw，可以继续再挖十年，说不定那会儿k35就来了 lol。如果现在p大于k32的盘就得不偿失了。谢谢你的回复。
